### PR TITLE
referral-data-model

### PR DESCRIPTION
**Pull request by somithajosephHSL**

### CODE REVIEW

**Comment by somithajosephHSL** [comment]
@WenYi-Healthshare : I have renamed the field to Organisation Identifier as per your comment. Please approve if you're happy with the change. 
@kngatai : Let me know if you have any comments.

**Comment by kngatai** [comment]
Looks good @somithajosephHSL

**Comment by somithajosephHSL** [comment]
Thanks @kngatai, Please approve, otherwise it will not let me merge the PR.

File: referralDataModel.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "receivedDate": "datetime",
+  "status": "string",
+  "referrersPriority": "string",
+  "reasonforReferral": "string",
+  "referredtoService": "string",
+  "referralSource": "string",
+  "referrer": {
+    "referrerHealthProviderCode": "string",
+    "assigningSourceIdentifier": "string",
+    "assigningOrganisationidentifier": "string",
```
Comment (wliuhealthnz) [inline referralDataModel.json:15]: assigningOrganisationIdentifier will be consistent

Comment (somithajosephHSL) [reply referralDataModel.json]: Good spotting, updated.

**Review by kngatai — APPROVED** [review]
Ok, found how we can do PR approvals..ha

Model looks good this is what will glue Command and Query sides together

**Pull request merged**